app/src/pages/inside/stepPage/modals/makeDecisionModal/elements/testItemDetails/testItemDetails.jsx

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import Parser from 'html-react-parser';
+import commentIcon from 'common/img/comment-inline.svg';
+import asteriskIcon from 'common/img/asterisk-inline.svg';
+import { ERROR_LOGS_SIZE } from 'pages/inside/stepPage/modals/makeDecisionModal/constants';
+import { StackTraceMessageBlock } from 'pages/inside/common/stackTraceMessageBlock';
+import { uniqueId } from 'common/utils';
+import { ItemHeader } from '../itemHeader';
+import styles from './testItemDetails.scss';
+
+const cx = classNames.bind(styles);
+
+export const TestItemDetails = ({
+  item,
+  mode,
+  selectItem,
+  logs,
+  eventsInfo,
+  suggestRs,
```

Comment:
Use the property that you need derectly here, in this case you need only `highlightedLogId`.